ERCS/ERC-XXXX.md

```diff
@@ -0,0 +1,130 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 8019
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.